### DISCUSSION TITLE
Does textinput support to hide sensitive information like secrets?

### DISCUSSION THREAD

**Discussion by vgjm** [charmbracelet/bubbles#136] [Q&A] [answered]
## Description
I am building a command line tool that collect some user inputs. Some of them are sensitive. Like secrets and tokens. I don't know how to hide these information when typing. Both hiding the input or showing some asterisks will be fine.
## Question
If there is a easy way that I don't know can achieve this or textinput just doesn't support this yet?

**Comment by muesli** [comment, marked as answer]
Yes, you can change the `EchoMode` to `EchoPassword` or `EchoNone` to achieve this.

**Reply by vgjm** [reply]
It's super helpful. Thanks a lot!
